tests/unit/igraph_attribute_combination_remove.c

```diff
@@ -0,0 +1,70 @@
+/*
+   IGraph library.
+   Copyright (C) 2021  The igraph development team <igraph@igraph.org>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include <igraph.h>
+#include "test_utilities.inc"
+
+void print_comb(igraph_attribute_combination_t *comb) {
+    int i;
+    igraph_attribute_combination_record_t *r;
+    for (i = 0; i < igraph_vector_ptr_size(&comb->list); i++) {
+        r = VECTOR(comb->list)[i];
+        if (r->name) {
+            printf("name: %s\n", r->name);
+        } else {
+            printf("name: NULL\n");
+        }
```

Comment:
Let's also print (and validate) the type here. It is not needed to convert the type into a string, just cast it to an integer and print that.